pkg/webhook/util/writer/atomic/atomic_writer.go

```diff
@@ -85,15 +85,18 @@ const (
 // The Write algorithm is:
 //
 //  1. The payload is validated; if the payload is invalid, the function returns
-//  2. The current timestamped directory is detected by reading the data directory
+//     2. The current timestamped directory is detected by reading the data directory
```

Comment:
Perhaps there is no need to change the descheduler, scheduler, and webhook packages.

Reply:
+1